### PR TITLE
[T3A3][T15-A4]Bartłomiej Woźniak

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by bartekwozniak93** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -55,19 +55,17 @@ public static boolean isValidAddress(String test) {
 
 	@Override
 	public String toString() {
-		return block.getBlock() + " " + street.getStreet() + " " + unit.getUnit() + " " + postalCode.getPostalCode();
+		return block.getBlock() + "," + street.getStreet() + "," + unit.getUnit() + "," + postalCode.getPostalCode();
 	}
 
 	@Override
 	public boolean equals(Object other) {
 		return other == this // short circuit if same object
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/data/person/Address.java]: Indentation problem?

File: src/seedu/addressbook/data/person/Block.java [hunk 2]
```diff
@@ -3,6 +3,10 @@
 public class Block {
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/data/person/Block.java]: Missing comments for the class.

**Comment by nishantbudhdev** [comment]
@bartekwozniak93 great work.

* Try regression testing before committing. This will help avoid some of the issues between your commits.
* As mentioned in tutorial, use a new branch for each activity. 

Some other comments are added. Please close the PR after reading them.

**Pull request closed without merging**